airbyte/caches/bigquery.py

```diff
@@ -38,8 +43,9 @@ class BigQueryCache(CacheBase):
     dataset_name: str = "airbyte_raw"
     """The name of the dataset to use. In BigQuery, this is equivalent to the schema name."""
 
-    credentials_path: str
-    """The path to the credentials file to use."""
+    credentials_path: str | None
```

Comment:
Pydantic in Python 3.9 doesn't like the `|` operator.

```suggestion
    credentials_path: Optiona[str]
```